src/models/demand.ts

```diff
@@ -33,6 +33,8 @@ export interface DemandResponse {
 export interface DemandPayload extends DemandResponse, Attachment {
   subtype: DemandSubtype
   binary_blob: Blob
+  latestTokenId: number
```

Comment:
as it looks we have not updated to be UUIDs and still incrementing?

Reply:
yep still incrementing integer on chain